Tasks/NuGetAuthenticateV1/taskJsonOverride.json

```diff
@@ -0,0 +1,43 @@
+{
+    "inputs": [
+        {
+            "name": "nuGetServiceConnections",
+            "type": "connectedService:ExternalNuGetFeed",
+            "label": "Service connection credentials for feeds outside this organization",
+            "required": false,
+            "helpMarkDown": "Comma-separated list of NuGet service connection names for feeds outside this organization/collection. For feeds in this organization/collection, leave this blank; the build’s credentials are used automatically.",
+            "properties": {
+                "EditableOptions": "False",
+                "MultiSelectFlatList": "True"
+            }
+        },
+        {
+            "name": "forceReinstallCredentialProvider",
+            "type": "boolean",
+            "label": "Reinstall the credential provider even if already installed",
+            "defaultValue": "false",
+            "helpMarkDown": "If the credential provider is already installed in the user profile, determines if it is overwritten with the task-provided credential provider. This may upgrade (or potentially downgrade) the credential provider."
+        },
+        {
+            "name": "workloadIdentityServiceConnection",
+            "label": "'Entra Workload ID-backed Azure DevOps user' Service Connection",
+            "helpMarkDown": "If this is set, feedUrl is required. All other inputs are ignored.",
+            "type": "connectedService:workloadidentityuser",
+            "required": false,
+            "properties": {
+              "EditableOptions": "False",
+              "MultiSelectFlatList": "False"
+            },
+            "groupName": "feedAuthentication"
```

Comment:
Do you need this?

Reply:
This was pulled from the PipAuthenticate PR, but it doesn't appear to be needed. I can remove it here since we only have a couple of input variables anyways